app/components/file-renderer/component.ts

```diff
@@ -0,0 +1,42 @@
+import { action, computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import config from 'ember-get-config';
+
+/**
+ * @module ember-osf
+ * @submodule components
+ */
+
+/**
+ * Render the provided url in an iframe via MFR
+ *
+ * Sample usage:
+ * ```handlebars
+ * {{file-renderer
+ *   download=model.links.download
+ *     width="800" height="1000" allowfullscreen=true}}
+ * ```
+ * @class FileRenderer
+ */
+export default class FileRenderer extends Component {
+    download: string;
+    width: string = this.width || '100%';
+    height: string = this.height || '100%';
+    allowfullscreen: boolean = this.allowfullscreen || true;
```

Comment:
camelCase?

Reply:
also, should be `&&`, so passing `false` isn't overwritten.

Reply:
I like camelCase here, but we should be careful about subtle API changes. Big breaking changes are easier to spot.

Reply:
yikes, booleans!

Reply:
Also, re: camelCase, this directly maps to iframe's `allowfullscreen` so maybe it should remain all lowercased?